gammapy/astro/darkmatter/spectra.py

```diff
@@ -153,6 +153,7 @@ def channel(self, channel):
             self._channel = channel
 
     def evaluate(self, energy, **kwargs):
+        """Evaluate the PrimaryFlux."""
```

Comment:
Shouldn't `PrimaryFlux` be given as a reference ? i.e. :

```suggestion
        """Evaluate the `~gammapy.astro.darkmatter.PrimaryFlux`."""
```

Reply:
Keeping in line with the other evaluate functions I don't think we should reference the class inside the class. But I have adapted it slightly

Reply:
Yes, I agree then !